paparazzi/paparazzi/src/main/java/app/cash/paparazzi/DeviceConfig.kt

```diff
@@ -445,6 +445,42 @@ data class DeviceConfig(
       released = "October 15, 2020"
     )
 
+    @JvmField
+    val PIXEL_6 = DeviceConfig(
+      screenHeight = 2400,
+      screenWidth = 1080,
+      xdpi = 409,
+      ydpi = 412,
```

Comment:
```suggestion
      xdpi = 406,
      ydpi = 411,
```

per the source of truth: https://android.googlesource.com/platform/tools/base/+/refs/heads/mirror-goog-studio-main/sdklib/src/main/resources/com/android/sdklib/devices/nexus.xml